packages/manager/apps/dedicated/client/app/billing/confirmTerminate/billing-confirmTerminate.html

```diff
@@ -1,218 +1,58 @@
-                    ></span
-                ></oui-message>
-            </div>
-
-            <h4
-                class="oui-header_4 mt-5"
-                data-translate="terminate_service_services_title"
-            ></h4>
-
-            <div class="text-center" data-ng-if="TerminateServiceCtrl.loading">
-                <oui-spinner></oui-spinner>
-            </div>
-
-            <div
-                class="table-responsive"
-                data-ng-if="!TerminateServiceCtrl.loading"
-            >
-                <table class="table table-hover">
-                    <thead>
-                        <tr>
-                            <td
-                                scope="col"
-                                data-translate="terminate_service_services_service"
-                            ></td>
-                            <td
-                                scope="col"
-                                data-translate="terminate_service_services_name"
-                            ></td>
-                            <td
-                                scope="col"
-                                data-translate="terminate_service_services_renew"
-                            ></td>
-                            <td
-                                scope="col"
-                                data-translate="terminate_service_services_frequency"
-                            ></td>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        <tr>
-                            <th
-                                scope="row"
-                                data-translate="{{ 'terminate_service_type_' + TerminateServiceCtrl.serviceInfos.serviceType }}"
-                            ></th>
-                            <td
-                                data-ng-bind="TerminateServiceCtrl.serviceInfos.domain"
-                            ></td>
-                            <td>
-                                <billing-renew-date
-                                    data-service-infos="TerminateServiceCtrl.serviceInfos"
-                                ></billing-renew-date>
-                            </td>
-                            <td>
-                                <billing-renew-label
-                                    data-service-infos="TerminateServiceCtrl.serviceInfos"
-                                ></billing-renew-label>
-                            </td>
-                        </tr>
-                    </tbody>
-                </table>
-            </div>
-
-            <div
-                class="mt-5"
-                data-ng-if="!TerminateServiceCtrl.loading &&
-                             TerminateServiceCtrl.serviceState !== 'suspending' &&
-                             TerminateServiceCtrl.serviceState !== 'suspended'"
+<oui-page-header
+    data-heading="{{:: 'billing_confirm_termination_resiliate' | translate:{
```

Comment:
you could safely

```suggestion
    heading="{{:: 'billing_confirm_termination_resiliate' | translate:{
```

Reply:
Habits 😄

Reply:
It's not like it's imperative, do whatever you want !